packages/react-notifications/hooks/useNotifications.ts

```diff
@@ -10,5 +22,7 @@ export const useNotifications = (group: string = "default") => {
     add: (children: ReactNode, options?: { autoDismiss?: number | false }) =>
       context.add(group, children, options),
     remove: (id: number) => context.remove(group, id),
+    clearGroup: (group: string) => context.clearGroup(group),
+    clearAllGroups: () => context.clearAllGroups(),
```

Comment:
we discussed to change `clearGroup(group)` to `clear()` and to remove the `clearAllGroups` for now, because at the moment, the hook is _always_ scoped to a group (even if you don't provide a group name it will default to "default").